pages/c/_cluster/monitoring/index.vue

```diff
@@ -0,0 +1,150 @@
+<script>
+import { mapGetters } from 'vuex';
+
+import InstallRedirect from '@/utils/install-redirect';
+
+import { NAME, CHART_NAME } from '@/config/product/monitoring';
+import { MONITORING } from '@/config/types';
+
+import LazyImage from '@/components/LazyImage';
+import ResourceGauge from '@/components/ResourceGauge';
+
+export default {
+  components: {
+    LazyImage,
+    ResourceGauge,
+  },
+
+  middleware: InstallRedirect(NAME, CHART_NAME),
+
+  data() {
+    return {
+      externalLinks: [],
+      grafanaSrc:    require('~/assets/images/white-grafana.svg'),
+      prometheusSrc: require('~/assets/images/white-prometheus.svg'),
+      resources:     [MONITORING.ALERTMANAGER, MONITORING.PROMETHEUSE],
+    };
+  },
+
+  computed: {
+    ...mapGetters(['currentCluster']),
+
+    accessibleResources() {
+      return this.resources.filter(resource => this.$store.getters['cluster/schemaFor'](resource));
+    },
+  },
+
+  mounted() {
+    this.externalLinks = [
+      {
+        iconSrc: this.prometheusSrc,
+        label:   'monitoring.overview.linkedList.alertManager.label',
+        link:    `/k8s/clusters/${ this.currentCluster.id }/api/v1/namespaces/cattle-monitoring-system/services/http:rancher-monitoring-alertmanager:9093/proxy`,
```

Comment:
@vincent99 This may not be the correct way to create these links. I know there was some talk about passing URLs in the chart values but I didn't see those params in chart themself yet. I can update this if that is ready obviously.